general/payment_acquirers/authorize.rst

```diff
@@ -1,81 +1,116 @@
-==================================
-How to get paid with Authorize.Net
-==================================
+=============
+Authorize.Net
+=============
 
-Authorize.Net is one of the most popular eCommerce payment platforms in North America.
-Unlike most of the other payment acquirers compatible with Odoo, 
-Authorize.Net can be used as `payment gateway <https://www.authorize.net/solutions/merchantsolutions/pricing/?p=gwo>`__ only.
-That way you can use the `payment processor or merchant <https://www.authorize.net/partners/resellerprogram/processorlist/>`__ that you like.
+`Authorize.Net <https://www.authorize.net>`__ is a United States-based online payment solution
+provider, allowing businesses to accept **credit cards**.
 
+.. image:: media/authorize-net.png
+   :align: center
+   :alt: Authorize.Net logo
 
-Create an Authorize.Net account
-===============================
+This :doc:`Payment Acquirer <payment_acquirers>` offers additional options that are not available
+for other Payment Acquirers, such as the ability to process your customer's payment after delivery.
```

Comment:
Just a detail but I would put the link on the second "Payment Acquirers" instead of the first one. The first one is Authorize.Net only, while the second is about all the payment acquirers. So, it would be more logical (to me) if the link was on "Payment Acquirers" (instead of payment acquirer).

Reply:
good idea :)